cloudpebble/settings.py

```diff
@@ -144,7 +145,7 @@
 )
 
 BOWER_INSTALLED_APPS = (
-    'https://github.com/krisk/Fuse.git#2c1560d763',
+    'https://github.com/krisk/Fuse.git',
```

Comment:
Is there any particular reason you changed it? If not, I'd change it back. If yes, specify the commit you want now; they've broken it once before.